sdk/monitor/azquery/CHANGELOG.md

```diff
@@ -3,12 +3,15 @@
 ## 0.2.1 (Unreleased)
 
 ### Features Added
+* Added `GetColumnIndex()` method to Table struct
```

Comment:
This needs to be updated.

Reply:
Thanks! Fixed in [34b4019](https://github.com/Azure/azure-sdk-for-go/pull/19374/commits/34b4019fe641923edad4b8d4e8ac5e4d9d804258)